addons/hr_presence/data/ir_cron.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record forcecreate="True" id="ir_cron_presence_control" model="ir.cron">
+            <field name="name">HR Presence: cron</field>
+            <field name="model_id" ref="hr.model_hr_employee"/>
+            <field name="state">code</field>
+            <field name="code">model._check_presence()</field>
+            <field eval="True" name="active" />
+            <field name="user_id" ref="base.user_root" />
+            <field name="interval_number">1</field>
+            <field name="interval_type">minutes</field>
```

Comment:
I had set it to 15 minutes. But apr wasn't happy because it was inconsistent and said that FP is "attaché" to this feature. I'll set it back to 15 minutes nonetheless
And it's intended to call the method each time you click on the presence menuitem, so its always accurate on the report view.

Reply:
On the other hand @odony is "attaché" to the performances on the SaaS platform.

1 hours is sufficient. No need to call this every 15 minutes during the whole week end ;).